clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -3583,24 +3585,44 @@ void populateCIRToLLVMConversionPatterns(mlir::RewritePatternSet &patterns,
 }
 
 namespace {
+
+using IRASMap = mlir::cir::AddressSpaceAttr::MapTy;
```

Comment:
I think `MeowTy` is a common pattern for member types in class. For free type names used in variable/parameter declarations, I prefer a clean CamelCase without suffix, because CamelCase is already the feature of types, no?

Btw, `IRASMap` copies its name from `clang::LangASMap` XD

Reply:
Makes sense!

Reply:
> I prefer a clean CamelCase without suffix

We should use the pattern that's used overall in LLVM despite our personal preferences, otherwise someone will need to fix this when we upstream this chunk of code.

Reply:
`IRASMap` -> `IRASMapTy`